src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -502,50 +502,86 @@ ExportQueryExecutionTrees::idToLiteral(const Index& index, Id id,
   auto datatype = id.getDatatype();
 
   switch (datatype) {
-    case WordVocabIndex:
-      return ad_utility::triple_component::Literal::literalWithoutQuotes(
-          index.indexToString(id.getWordVocabIndex()));
+    case WordVocabIndex: {
+      auto lit = getLiteralOrIriFromWordVocabIndex(index, id);
+      return getLiteralOrNullopt(lit);
```

Comment:
```suggestion
      return getLiteralOrNullopt(getLiteralOrIriFromWordVocabIndex(index, id));
```
No copy + no explicit `std::move()` call.